README.md

```diff
@@ -11,6 +11,9 @@ This is the backend client of Bridge-In-Tech.
 
 Please read our [Contributing Guidelines](.github/CONTRIBUTING.md), [Code of Conduct](.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](.github/REPORTING_GUIDELINES.md) thoroughly.
 
+## Setup
+To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup) wiki page.
+
```

Comment:
This looks good now, @mahakbansal2019. I'm happy to approve this. We just need to wait for one more reviewer to approve the PR before it can be merged. cc @anitab-org/coding-team or @anitab-org/bridgeintech-maintainers .

Thanks again for your contribution and patience in responding to each requested changes 😉